src/golang/lib/job/k8s.go

```diff
@@ -140,7 +140,7 @@ func (j *k8sJobManager) Launch(ctx context.Context, name string, spec Spec) JobE
 
 	containerRepo, err := mapJobTypeToDockerImage(spec, launchGpu)
 	if err != nil {
-		return systemError(err)
+		return userError(err)
```

Comment:
I think this should be a user error; it catches the k8s <> sqlite compatibility issue